modules/tools/decode_video/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//tools:cpplint.bzl", "cpplint")
```

Comment:
Generally we only put python code to `modules/tools`.
Plz move this to either `modules/drivers` or `modules/data`.

Reply:
done